src/SearchRoom/SearchRoom.css

```diff
@@ -0,0 +1,42 @@
+  h1 {
+    margin:0;
+  }
+  
+  .search-room-title { 
+    font-size: var(--title-font-size);
+    font-weight: bold;
+  }
```

Comment:
Just noticed this looks a little different to .view-room-title, could you either add the following css to this class or alternatively create a shared class in Main.css so the styling matches?
` padding-bottom: 8px;
  padding-top: 8px;`